src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -84,8 +84,10 @@ public function purgeAll()
      */
     private function getPurgeHeaders()
     {
+        $host = parse_url($this->configResolver->getParameter('http_cache.purge_servers')[0], PHP_URL_HOST);
+
         $headers = [
-            'Host' => empty($_SERVER['SERVER_NAME']) ? parse_url($this->configResolver->getParameter('http_cache.purge_servers')[0], PHP_URL_HOST) : $_SERVER['SERVER_NAME'],
+            'Host' => $host ?: $_SERVER['SERVER_NAME'],
```

Comment:
Looks good, only think that comes to mind is PHP notice when this fallback server var is sometimes not set, suggestions?

Reply:
Can you please check https://github.com/ezsystems/ezplatform-http-cache/pull/119/commits/f63f51e8a351559a34f15136584bef3f173f02bf?

Reply:
I think you misunderstood me, my point was about ` $_SERVER['SERVER_NAME']` now being used w/o check if it is set first.

To avoid having to adapt tests I propose you revert f63f51e
And that should be ok, my comment can be ignored, there is anyway nothing to fallback to form SERVER_NAME unless we hard code a last fallback to `localhost` like we did [before](https://github.com/ezsystems/ezplatform-http-cache/blob/v0.8.0/src/PurgeClient/VarnishPurgeClient.php#L46).